model/resource/host.yaml

```diff
@@ -70,3 +70,46 @@ groups:
           The version string of the VM image or host OS as defined in
           [Version Attributes](README.md#version-attributes).
         examples: ['0.1']
+  - id: host.cpu
+    prefix: host.cpu
+    type: resource
+    brief: >
+      A host's CPU information
+    attributes:
+      - id: vendor.id
+        requirement_level: opt_in
+        type: string
+        brief: >
+          Processor manufacturer identifier. A maximum 12-character string.
```

Comment:
Where does this limitation of 12-character string come from?
If that's a specific format, can we link to the specification from here?

Reply:
From what I found it comes from the fact that it is stored directly in 3 specific CPU's registers: https://en.wikipedia.org/wiki/CPUID#EAX=0:_Highest_Function_Parameter_and_Manufacturer_ID. So 3x4bytes=12 bytes=12 ASCII characters.

This information is coming directly from the [kernel](https://www.kernel.org/doc/Documentation/filesystems/proc.txt) and unfortunately I cannot find any well defined section for it within the kernel docs: https://www.kernel.org/doc/html/latest/search.html?q=cpuinfo. It seems that some docs where added recently for cpuinfo flags but this does not help a lot here: https://lwn.net/ml/linux-kernel/20200610200701.16757-1-kyung.min.park@intel.com/.

I will look again though in case I can find something more specific.

Reply:
@ChrsMark Maybe we could link this one: https://wiki.osdev.org/CPUID ?

Reply:
Added. Resolving this as complete.